cfme/configure/configuration/__init__.py

```diff
@@ -515,6 +519,49 @@ def update(self):
         self.appliance.server_details_changed()
 
 
+class VMwareConsoleSupport(Updateable, Pretty, Navigatable):
+    """
+    This class represents the "VMware Console Support" section of the Configuration page.
+    Note this is to support CFME 5.8 and beyond functionality.
+
+    Args:
+        console_type:  One of the following strings 'VMware VMRC Plugin', 'VNC' or 'VMware WebMKS'
+
+    Usage:
+
+        vmware_console_support = VMwareConsoleSupport(console_type="VNC")
+        vmware_console_support.update()
+
+    """
+    vmware_console_form = Form(
+        fields=[
+            ('console_type', AngularSelect("console_type")),
+        ]
+    )
+    pretty_attrs = ['console_type']
+
+    CONSOLE_TYPES = ['VMware VMRC Plugin', 'VNC', 'VMware WebMKS']
+
+    def __init__(self, console_type, appliance=None):
+        if console_type not in VMwareConsoleSupport.CONSOLE_TYPES or console_type is None:
```

Comment:
I like where your mind is here, being 'safe' - but the check of console_type is None is redundant. If it is `None`, the first statement will be false, as the CONSOLE_TYPES list doesn't include a 'None' element.